### ISSUE TITLE
Test failure on non-Linux systems

### ISSUE THREAD

**Issue by eserte** [petdance/html-tidy5#1]
The test suite fails on freebsd system, because apparently "Linux" is expected but "FreeBSD" was generated. For example:
```
#   Failed test '$tidy->clean("") returns empty HTML document'
#   at t/clean.t line 21.
#                   '<!DOCTYPE html>
# <html>
# <head>
# <meta name="generator" content="HTML Tidy for HTML5 for FreeBSD version 5.6.0">
# <title></title>
# </head>
# <body>
# </body>
# </html>
# '
#     doesn't match '(?^:<!DOCTYPE html>
# <html>
# <head>
# <meta name="generator" content="HTML Tidy for HTML5 for Linux version \d+\.\d+\.\d+">
# <title></title>
# </head>
# <body>
# </body>
# </html>
# )'
# Looks like you failed 1 test of 3.
t/clean.t ............ 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests 
```
(and more of these)

**Comment by eserte** [comment]
Something similar may happen also on Debian/stretch, where apparently only tidy 5.2.0 is available, not the expected 5.6.0:
```
    #          got: '<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2//EN">
    # <html>
    # <head>
    # <meta name="generator" content="HTML Tidy for HTML5 for Linux version 5.2.0">
    # <title>æ¥æ¬èªã®ãã ãã¼ã¸</title>
    # </head>
    # <body>
    # <p>Unicodeãå¥½ãã§ãã?</p>
    # </body>
    # </html>
    # '
    #     expected: '<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2//EN">
    # <html>
    # <head>
    # <meta name="generator" content="HTML Tidy for HTML5 for Linux version 5.6.0">
    # <title>æ¥æ¬èªã®ãã ãã¼ã¸</title>
    # </head>
    # <body>
    # <p>Unicodeãå¥½ãã§ãã?</p>
    # </body>
    # </html>
    # '
    # Looks like you failed 1 test of 3.

#   Failed test 'Try send bytes to clean method.'
#   at t/unicode.t line 65.
# Looks like you failed 2 tests of 2.
t/unicode.t .......... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/2 subtests 
```

**Comment by petdance** [comment]
This is fixed in 0.02.  Thanks.
